matrix.json

```diff
@@ -33,6 +33,14 @@
                         "type": "crypto",
                         "ticker": "BCH"
                     },
+                    {
+                        "type": "crypto",
+                        "ticker": "ETH"
+                    },
+                    {
+                        "type": "crypto",
+                        "ticker": "ERC20"
+                    },
```

Comment:
As you've noticed, this is not how the schema works. The Matrix needs to support users selecting a _specific_ coin name from the "currencies" in the drop down menu at: https://walletmatrix.app/matrix. Yes this can become large and unwieldy, but then vendors need not include _all_ their coins: The addition of currencies into the schema is merely incidental seeing is really all about Bitcoin. I have made a note to look at deprecating this for schema v4.

Reply:
On the other hand, many people ask for those shitcoins and ERC20 should be a valid option as some wallets allow the user to add their own ERC20 tokens, which makes a lot of sense.

Being a Maxi myself I would be fine with dropping this but it's political and if the goal is to develop a standard by which all wallet providers can describe their feature sets, it should not be pruned. There are products out there that very explicitly do not touch shitcoins and I would like to filter for those. `feature: btcOnly`? Or maybe `coinsSupported < 2`? A numerical `coinsSupported` would make it easier for those shit-coin casinos to get listed without having to keep their selection of the day updated.

Reply:
I removed the ERC20 and added other details

Reply:
@Giszmo Until yesterday, the goal of these schema was to support consumption into WM. Period. Once more examples of `matrix.json` exist in vendor's public repos, then we can have a conversation as to a). the best method by which these files might be consumed (individually, or through a WM/WS managed API, file or service and b). what a v4 schema might look like. 

ATM the schema is a standard format intended purely for consumption by WM. Eventual consumers (via API for example) needn't be constrained by what is essentially an _internal_ format. Instead and by consensus, a v1,2 (N) API can be designed and built that better suits how third-parties might want to consume it.

I'm super keen to hear further ideas for improvement to the schema - but I'm not seeing right now how the schema _itself_ is going to be useful for anything other than WM.

Question: Are you an official contributer/maintiner to MyCelium for Android? Or do you know them? `matrix.json` should exist in the _vendor_ repo. These decentrica-repos are only a temporary measure until more vendors include it in their own repos.

Reply:
I worked for Mycelium for 5 years but quit in June to work on WS. They would probably not maintain the matrix file in its current state. With integration in WS and some polished and more popular WM ... maybe.